app/web/features/profile/view/UserOverview.test.tsx

```diff
@@ -0,0 +1,119 @@
+import { render } from "@testing-library/react";
+import { HostingStatus, MeetupStatus } from "proto/api_pb";
+import wrapper from "test/hookWrapper";
+import { addDefaultUser } from "test/utils";
+
+import {
+  COMMUNITY_STANDING,
+  hostingStatusLabels,
+  meetupStatusLabels,
+  VERIFICATION_SCORE,
+} from "../constants";
+import { ProfileUserProvider } from "../hooks/useProfileUser";
+import UserOverview from "./UserOverview";
+
+describe("UserOverview", () => {
+  beforeEach(() => {
+    addDefaultUser();
+  });
+
+  describe("when user is missing", () => {
+    it("doesn't fail", () => {
+      render(<UserOverview />, { wrapper });
+    });
+  });
```

Comment:
This is a bit of a false test as this component has a dependency on `ProfileUserProvider` existing up the tree, so I don't think this test should exist to give the false impression that the component can be used without a provided user...

Reply:
you're right. I'll remove it